src/legacy/core_plugins/kibana/public/management/sections/objects/lib/is_same_query.ts

```diff
@@ -17,9 +17,9 @@
  * under the License.
  */
 
-import { parseQuery } from '.';
+import { parseQuery } from './parse_query';
 
-export const isSameQuery = (query1, query2) => {
+export const isSameQuery = (query1: any, query2: any) => {
```

Comment:
I cannot see we use it. I would rather remove the code. We always can find removed code in the git history.

Reply:
Yea, only usage is in a TODO (🙈) 

https://github.com/elastic/kibana/blob/7df981fbf8f81e1df205082a883a74ca99839e22/src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/objects_table.js#L240-L242

I'll remove it for now